### ISSUE TITLE
[FEATURE] Add exclude option in re_data CLI

### ISSUE THREAD

**Issue by FrankTub** [re-data/re-data#335]
**Tell us about the problem you're trying to solve**

In my current environment I have very few test failures or anomalies, but a lot of schema changes. This makes sense since we are only just starting out. It would be helpful if we could have some more engagement with business users to solve data quality issues. But when we now generate re_data output and host in a S3 Bucket in AWS end users can't find what they are looking for, since they will see a lot of schema changes (which are not relevant to them) and very few failed tests (which are very relevant to them). I would like to be able to generate re_data's overview with the ability to exclude schema changes.

**Describe the solution you’d like**

In `re_data` cli I would like to have an `--exclude` flag to exclude schema_changes (and possible things like tests or anomalies). This does not yet [exist](https://docs.getre.io/latest/docs/reference/cli/overview).

**Describe the alternative you’ve considered or used**

Solely alerting users (where this exclude option is available). But I would prefer to have users access the UI since I think this gives more context to users what is going on.

**Comment by mateuszklimek** [comment]
@FrankTub Thanks for creating the issue! I wonder about one thing, I feel that our alerts UI needs alert_type select option.
Where you will be able to choose between (schema_change, anomaly, test) failures to show. I wonder if that would solve issues for you?

You would also be able to share the URL with `?type=anomaly` added so that it would contain only info about anomalies and not schema changes.

Let me know :)

**Comment by FrankTub** [comment]
Yeah it's mostly based on the alerts UI that I opened this issue. Having the option to choose between `schema_change`, `anomaly` and `test` on that page would really help. Would be even more awesome if we could select by default `anomaly` and `test`.
